### PR TITLE
Remove unnecessary print statements

### PR DESCRIPTION
**Pull request by dilanSachi**
## Purpose
Remove print statements in client startup
## Examples
N/A
## Checklist
- [ ] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: kafka-native/src/main/java/org/ballerinalang/messaging/kafka/utils/KafkaConstants.java [hunk 1]
```diff
@@ -34,7 +34,7 @@ private KafkaConstants() {
     public static final int DURATION_UNDEFINED_VALUE = -1;
 
     // Kafka log messages
-    public static final String SERVICE_STARTED = "[ballerinax/kafka] started kafka listener ";
+    public static final String SERVICE_STARTED = "[ballerinax/kafka] started kafka listener for topic/s ";
```
Comment (aashikam) [inline kafka-native/src/main/java/org/ballerinalang/messaging/kafka/utils/KafkaConstants.java:37]: How about "Kafka listener started for topics: " WDYT? 
Also WDYT @ThisaruGuruge

Comment (dilanSachi) [reply kafka-native/src/main/java/org/ballerinalang/messaging/kafka/utils/KafkaConstants.java:37]: sounds much better

Comment (aashikam) [reply kafka-native/src/main/java/org/ballerinalang/messaging/kafka/utils/KafkaConstants.java:37]: That would be more consistent with rabbitmq https://ballerina.io/learn/by-example/rabbitmq-consumer.html

Comment (ThisaruGuruge) [reply kafka-native/src/main/java/org/ballerinalang/messaging/kafka/utils/KafkaConstants.java:37]: +1

Comment (dilanSachi) [reply kafka-native/src/main/java/org/ballerinalang/messaging/kafka/utils/KafkaConstants.java]: Updated now

**Review by aashikam — APPROVED** [review]

**Review by ThisaruGuruge — APPROVED** [review]

**Pull request merged**